src/services/api/registration/registration.service.ts

```diff
@@ -45,6 +48,13 @@ export class RegistrationService {
         LogContext.COMMUNITY
       );
     }
+
+    if (!agentInfo.authId?.trim()) {
```

Comment:
trim should not be needed

Reply:
Ideally, even strings are not needed, but typescript can not handle UUID natively, so we have to use horrible and inefficient way of dealing with strings, instead of 16-byte :) But I will recheck why there is a trim.